packages/x-components/src/x-modules/search/store/emitters.ts

```diff
@@ -34,8 +35,11 @@ export const searchEmitters = createStoreEmitters(searchXStoreModule, {
         totalResults: state.totalResults
       };
     },
-    filter: (newValue, oldValue) =>
-      newValue.status !== oldValue.status && oldValue.status === 'loading' && !!newValue.request
+    filter: (newValue, oldValue) => {
+      return (
+        newValue.status === oldValue.status && oldValue.status === 'loading' && !!newValue.request
```

Comment:
Now with this change, we are seeing a no results page momentarily before the filters are removed

https://user-images.githubusercontent.com/77450928/230880476-9572835e-94bf-490a-aa56-14f1e5fa308d.mov

Are we sure we want it to behave this way?